src/app/layout.tsx

```diff
@@ -36,7 +42,14 @@ export default function RootLayout({
       </head>
       <body className="bg-neu-0">
         <GoogleAnalytics />
-        {children}
+        <NewFontsStyleTag />
+        <ThemeProvider attribute="class">
+          <MenuProvider>
+            <div className="isolate bg-neu-0 text-neu-900 antialiased">
+              {children}
+            </div>
```

Comment:
Let's move it back to where it was as a sibling of footer and navbar.

```suggestion
            {children}
```